.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build and push container
+
+on:
+  push:
+    branches:
+      - '1.8.6'
```

Comment:
This branch has to match the branch in GitHub. It's no big deal -- I'll create a 1.8.6 branch and you can propose a PR against that one.